File: 168.ExcelSheetColumnTitle/excel.cc
#include <string>
using namespace std;
class Solution {
public:
    string convertToTitle(int n) {
        string ans = "";
        while (n > 0) {
            n -= 1;
            ans = char((n % 26) + 'A') + ans;
            n /= 26;
        }
        return ans;
    }
};

File: 168.ExcelSheetColumnTitle/excel.py
class Solution(object):
    def convertToTitle(self, n):
        """
        :type n: int
        :rtype: str
        """
        ans = ''
        while n > 0:
            n -= 1
            ans = chr(n % 26 + ord('A')) + ans
            n /= 26
        return ans

if __name__ == '__main__':
    for i in range(1, 100):
        print i, Solution().convertToTitle(i)
